MANIFEST.in

```diff
@@ -0,0 +1 @@
+recursive-include src/qtt/exampledata *
```

Comment:
Missing newline at the end

Reply:
Also, I raised a question about this issue, why we need the example data in the non editable installation because we also don't install the ipython notebooks that are using them. It is strange that production code is using some data from example data.

Reply:
If and only if we need to install some data, we should be selective and not installing the whole example data directory.

Reply:
Addressed the newline. The example datasets are part of the code and of the runnable jupyter examples. If we want to change that behavior I suggest we discuss are make a new PR.